NumLib/TimeStepping/Algorithms/DummyTimeStepping.h

```diff
@@ -0,0 +1,70 @@
+/**
+ * \author Norihiro Watanabe
+ * \date   2012-08-03
+ *
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#ifndef DUMMYTIMESTEPPING_H_
+#define DUMMYTIMESTEPPING_H_
+
+#include <vector>
+#include <boost/property_tree/ptree.hpp>
+
+#include "ITimeStepAlgorithm.h"
+
+namespace NumLib
+{
+
+/**
+ * \brief Fixed time stepping algorithm
+ *
+ * This algorithm is meant to perform exactly one timestep.
+ * To be used e.g. for time-invariant processes in a time stepping loop
+ */
+class DummyTimeStepping : public ITimeStepAlgorithm
```

Comment:
I would rename it to `SingleStepTimeAlgorithm` or something similar, repeating the behaviour described in the documentation.
A dummy variable/code/class has in contrary no behaviour at all (in my understanding).